www/src/scss/viewer/shabad-deck/_shabad-deck.scss

```diff
@@ -4,12 +4,16 @@
     background-size: cover;
     height: unset;
     padding-bottom: 10px;
+    position: fixed;
+    overflow-y: scroll;
```

Comment:
@theamanjs veerji, everything looks fine here. Just a minor change, can you switch up the order of CSS properties. overflow-y should be before position.

Reply:
@Gauravjeetsingh thank you for your feedback veerji, I'm doing it right now :smile:

Reply:
@Gauravjeetsingh veerji, I have made the changes.